pytest-selenium/tests/test_attribution_pom.py

```diff
@@ -0,0 +1,127 @@
+
+from time import sleep
+
+import pytest
+from selenium import webdriver
+
+from pages.content import Content
+from tests import markers
+
+
+class TestCitationtexturl():
+
+#c476303 attribution section is initially collapsed, expands when clicked
+    @markers.test_case("C476303")
+    @markers.parametrize("book_slug,page_slug", [("college-physics", "preface")])
+    @markers.nondestructive
+
+    def test_attribution_initial_status(self, selenium, base_url, book_slug, page_slug):
+        
+#Given - the page {base_url}/books/college-physics/pages/preface
```

Comment:
We always indent 4 spaces even if it is a comment. I'll show you how to not worry about formatting soon. We need to install the pre-commit hooks and before every commit you'll be able to run checks against your code to make sure you have no linting or formatting issues.

Reply:
Haha...I intentionally deleted the spaces for the given-when-then lines so that the scenario stands out and easily readable...I will change that though

Reply:
Updated the comments to indent 4 spaces